viewer/index.html

```diff
@@ -0,0 +1,224 @@
+            <div class="mt-3">
+              <ul class="nav list-group list-group-flush wl__sidebar-feature-name-list"></ul>
+            </div>
+          </div>
+        </div>
+
+        <div class="main">
+          <div class="wl-table-wrap d-none" id="table-content">
+            <div class="wl-table">
+              <div class="wl-table-heading">
+                <div class="wl-table-row wl-table-row--bottom-shadow">
+                  <div class="wl-table-head">Feature</div>
+                  <div class="wl-table-head">Frequent items</div>
+                  <div class="wl-table-head">Inf. feature type</div>
+                  <div class="wl-table-head text-end">Total count</div>
+                  <div class="wl-table-head text-end">Null fraction</div>
+                  <div class="wl-table-head text-end">Est. unique values</div>
+                  <div class="wl-table-head">Data type</div>
+                  <div class="wl-table-head text-end">Data type count</div>
+                  <div class="wl-table-head text-end">Mean</div>
+                  <div class="wl-table-head text-end">Std. dev</div>
+                  <div class="wl-table-head text-end">Min</div>
+                  <div class="wl-table-head text-end">First quantile</div>
+                  <div class="wl-table-head text-end">Median</div>
+                  <div class="wl-table-head text-end">Third quantile</div>
+                  <div class="wl-table-head text-end">Max</div>
+                </div>
+              </div>
+              <ul class="wl-table-body wl__table-body"></ul>
+            </div>
+          </div>
+
+          <div class="wl-table__text-message" id="table-message">
+            <p>Please select and load a JSON profile from your local drive.</p>
+          </div>
+
+          <div class="wl-property-panel">
+            <button type="button" class="wl-property-panel__button btn-close" aria-label="Close"></button>
+            <h5 class="wl-property-panel__title"></h5>
+            <div>
+              <table class="wl-property-panel__table">
+                <thead class="wl-property-panel__table-head">
+                  <tr class="wl-property-panel__table-tr">
+                    <th class="wl-property-panel__table-th">Item</th>
+                    <th class="wl-property-panel__table-th wl-property-panel__table-th-profile"></th>
+                  </tr>
+                </thead>
+                <tbody class="wl-property-panel__frequent-items">
+                  <!-- CONTENT IS GENERATED -->
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- Bootstrap core JavaScript
+    ================================================== -->
+    <!-- Placed at the end of the document so the pages load faster -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
do we need jquery for this?

Reply:
It's used in the JS -- sort of our replacement interactivity (since there is no react)

Reply:
You guys probably saw this already, but it looks like there's a library for generating static pages with react: https://github.com/react-static/react-static

But yea, jquery is probably a bit easier to work with than vanilla js in this case (especially if we're trying to keep dependencies to a minimum).

Reply:
We were discussing react-static, but because we already had something in vanilla and in order to keep dependencies at the minimum, we decided to continue without vanilla.